templates/repo/issue/view_content/comments.tmpl

```diff
@@ -372,7 +372,7 @@
 				<div class="timeline-item event">
 					{{if .OriginalAuthor}}
 					{{else}}
-					<a class="timeline-avatar"{{if gt .Poster.ID 0}} href="{{.Poster.HomeLink}}"{{end}}>
+					<a class="timeline-avatar{{if or (and (eq .Review.Type 1) (or .Content .Attachments)) (eq .Review.Type 2) (eq .Review.Type 3)}} timeline-avatar-offset{{end}}"{{if gt .Poster.ID 0}} href="{{.Poster.HomeLink}}"{{end}}>
```

Comment:
What's this used for? Looks difficult to understand ....

And the line above might be rewritten as `{{if not .OriginalAuthor}}` , instead of `{{else}}`

Reply:
Comment added.